packages/neo-one-smart-contract-lib/src/index.ts

```diff
@@ -1,4 +1,3 @@
 // tslint:disable-next-line export-name
 export { NEP5Token } from './NEP5Token';
-export { DesignatedCaller } from './ownership/DesignatedCaller';
```

Comment:
This commit isn't based off master - please make sure your PRs/commits are based off master.